src/ripple/core/ThreadEntry.h

```diff
@@ -17,8 +17,8 @@
 */
 //==============================================================================
 
-#ifndef RIPPLE_CORE_REPORT_UNCAUGHT_EXCEPTION_H_INCLUDED
-#define RIPPLE_CORE_REPORT_UNCAUGHT_EXCEPTION_H_INCLUDED
+#ifndef RIPPLE_CORE_THREAD_ENTRY_H_INCLUDED
+#define RIPPLE_CORE_THREAD_ENTRY_H_INCLUDED
 
 #include <ripple/basics/Log.h>
 #include <boost/coroutine/exceptions.hpp> // forced_unwind exception
```

Comment:
But when you do remove the includes, add some of them back to ThreadEntry.cpp.  I'm thinking `<iostream>`, `<exception>`, `<ripple/basics/Log.h>` and `<string>` need to be seen by ThreadEntry.cpp.